Add episode 01 (Assigning values to variables).

mkcor opened the pull request:
Dear @maikia,

I have adapted the episode on [Python Fundamentals](https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html) with some slight inspiration from [Variables and Assignment](http://swcarpentry.github.io/python-novice-gapminder/02-variables/index.html).

Line 155, I'm referring to your episode about functions; maybe we could add a link to it afterwards, when you submit it.

Cheers,
Marianne

mkcor committed:
Add episode 01

mkcor committed:
Add figures

mkcor committed:
Link to episode and document its provenance

maikia commented on _episodes/01-variables.md:
```diff
@@ -0,0 +1,356 @@
+## Using Built-in Functions
+
+To carry out common tasks with data and variables in Python,
+the language provides us with several built-in functions.
```
maybe worth adding a short sentence on what `built-in` means?

mkcor replied:
I think that the word 'built-in' and the sentence 'the language provides us with several built-in functions' are self-descriptive (I mean, in plain English). But I'll add a note (to my personal 'instructor notes') to emphasize this when I speak (something along the lines of "they are *built into* the language itself, which means they are readily available for use, as opposed to functions which belong to specific collections, which need importing -- we'll touch on this later in the lesson"). What do you think?

maikia commented on _episodes/01-variables.md:
```diff
@@ -0,0 +1,356 @@
+
+When we want to make use of a function, which is referred to as _calling_ a
+function, we follow its name by parentheses. The parentheses are important:
+If you leave them off, the function doesn't actually run.
```
worth showing them what happens and that it's not an error, though function is not called

mkcor replied:
Done at 38d72de9c35.

maikia commented on _episodes/01-variables.md:
```diff
@@ -0,0 +1,356 @@
+We can display multiple things at once using only one `print` call:
+
+~~~
+print(patient_id, 'weight in kilograms:', weight_kg)
```
just a suggestion: how about also showing them how to print f-string? it's not much more complex than what you already show and very useful also for other things?

mkcor replied:
I have thought about it, I hesitated, and I thought I shouldn't add any cognitive load... But, since you're suggesting, I'm happy to add it, specifying that we're seeing this just in passing (just so they've heard of it). Thanks for the suggestion!

mkcor replied:
Done at 8d9dbaa043.

maikia commented on _episodes/01-variables.md:
```diff
@@ -0,0 +1,356 @@
+---
+title: Assigning values to variables
+teaching: 10
```
this renders to a very wide table, perhaps we could put it row under row? instead as columns?

mkcor replied:
Right; would you happen to know the syntax for this? :no_mouth:

maikia replied:
lol :-) not at all. I have actually never wrote my tutorial in .md so we will have a laugh when you review mine ;-)

Feel free to ignore, we will use the documentation (and not github display) for it anyway in which case it's just used for template and does not display as a table

mkcor replied:
Haha, ok, no worries!

maikia commented on _episodes/01-variables.md:
```diff
@@ -0,0 +1,356 @@
+~~~
+{: .output}
+
+> ## Variables as Sticky Notes
```
why to change the formatting from here on?

mkcor replied:
Oh, it comes from: https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#variables-as-sticky-notes "Variables as Sticky Notes" is a 'callout' section and the rest are challenges (see https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#check-your-understanding and so on).

Basically, I left the original formatting... I think it doesn't hurt, but I should probably copy SWC's CSS files for this render as expected.

maikia replied:
great. For now it just displays as a shifted paragraph so I was confused :-)

maikia reviewed (approved):
Thanks @mkcor , LGTM
Few minor comments but feel free to ignore them if you don't agree

mkcor committed:
Include a note about f-strings

mkcor committed:
Show output of print without parentheses

mkcor commented:
@maikia thank you so much for the attentive review! It really benefits the content to have this second pair of eyes. I have addressed your comments; if nothing else comes up, I will merge tomorrow. Thanks again!

maikia reviewed (approved):
LGTM, thanks @mkcor

mkcor commented:
Thanks again, @maikia! I'll merge now then.
